pyproject.toml

```diff
@@ -20,7 +20,7 @@ maintainers = [
 description = "Package to generate computational unstructured meshes for ocean simulation."
 license = {file = "LICENSE"}
 readme = "README.md"
-requires-python = '>=3.9, <3.14' # 3.8 -> scipy
+requires-python = '>=3.9, <3.15' # 3.8 -> scipy
```

Comment:
We need to also enable tests for newer versions of Python (all supported versions) defined in `.github/workflows`